src/main/kotlin/calculator/Calculator.kt

```diff
@@ -0,0 +1,29 @@
+package calculator
+
+import java.lang.IllegalArgumentException
+
+enum class Calculator(val symbol: String) {
+    ADD("+") {
+        override fun operation(a: Int, b: Int): Int = a + b
+    },
+    SUBTRACT("-") {
+        override fun operation(a: Int, b: Int): Int = a - b
+    },
+    MULTIPLY("*") {
+        override fun operation(a: Int, b: Int): Int = a * b
+    },
+    DIVIDE("/") {
+        override fun operation(a: Int, b: Int): Int = a / b
+    };
+
+    abstract fun operation(a: Int, b: Int): Int
```

Comment:
잘 구현해 주셨네요 👍 👍 

다만 아래와 같이 구현하는 방법도 있으니 참고 바랍니다.

```suggestion
enum class Calculator(
    val symbol: String,
    private val operation: (Int, Int) -> Int,
) {
    ADD("+", { a, b -> a + b }),
    SUBTRACT("-", { a, b -> a - b }),
    MULTIPLY("*", { a, b -> a * b }),
    DIVIDE("/", { a, b -> a / b }),
    ;

    fun operate(a: Int, b: Int): Int {
        return this.operation.invoke(a, b)
    }
```